### PR TITLE
Add command for checking whether an object has a type by comparing typenames

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Targets #441

Please review. The function names may not be appropriate, thought.

### CODE REVIEW

File: src/objects/object.c [hunk 1]
```diff
@@ -548,6 +548,34 @@ ws_object_is_instance_of(
...
+    struct ws_object* self,
+    const char* type_name
+) {
+    if (unlikely(!self || !type_name)) {
+        return false;
+    }
+
+    ws_object_lock_read(self);
+
+    ws_object_type_id* curtype = self->id;
```
Comment (neithernut) [inline src/objects/object.c:561]: Objects don't change their type. If they do, we are probably in serious trouble.
Anyways, you can unlock after this line.

File: src/command/object.c [hunk 2]
```diff
@@ -43,6 +43,33 @@
...
+    union ws_value_union* args
+) {
+    if ((args[0].value.type != WS_VALUE_TYPE_OBJECT_ID) ||
+            (args[1].value.type != WS_VALUE_TYPE_STRING)) {
+        return -EINVAL;
+    }
+
+    struct ws_object* o = ws_value_object_id_get(&args[0].object_id);
+
+    struct ws_value_string *v_type_name = &args[1].string;
```
Comment (manuelmessner) [inline src/command/object.c]: This variable can be omitted

**Comment by neithernut** [comment]
ACk for squashing.

**Comment by matthiasbeyer** [comment]
Done.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**